pkg/kopia/snapshot/snapshot.go

```diff
@@ -101,7 +104,7 @@ func reportSnapshotStatus(ctx context.Context, snapshotStartTime time.Time, mani
 
 // DeleteSnapshot deletes Kopia snapshot with given manifest ID
 func DeleteSnapshot(ctx context.Context, backupID, path, password string) error {
-	rep, err := OpenRepository(ctx, defaultConfigFilePath, password, pullRepoPurpose)
+	rep, err := repository.OpenRepository(ctx, kopia.defaultConfigFilePath, password, pullRepoPurpose)
```

Comment:
The method can be renamed to `Open` so that on import it becomes `repository.Open()`